src/mozanalysis/experiment.py

```diff
@@ -1354,11 +1430,11 @@ def _build_analysis_window_subset_query(
         in "the standard format" for a single analysis window.
         """
         return f"""
-            SELECT * EXCEPT (client_id, analysis_window_start, analysis_window_end)
+            SELECT * EXCEPT ({self.experimental_unit.value}, analysis_window_start, analysis_window_end)
             FROM {self.fully_qualified_table_name}
             WHERE analysis_window_start = {analysis_window.start}
             AND analysis_window_end = {analysis_window.end}
-        """
+        """  # noqa: E501
```

Comment:
Could you just split the `EXCEPT ( ... )` into multiple lines? I usually try to avoid `noqa` whenever possible, though I understand it can be a bit annoying so not a big deal if splitting the string introduces other issues.

Reply:
Done